.devcontainer/Dockerfile

```diff
@@ -36,12 +35,6 @@ RUN git clone -b v${KUBENS_VERSION} --single-branch https://github.com/ahmetb/ku
     && ln -s /opt/kubectx/kubectx /usr/local/bin/kubectx \
     && ln -s /opt/kubectx/kubens /usr/local/bin/kubens
 
-# Install Octant
-RUN curl -Lo octant.tar.gz https://github.com/vmware-tanzu/octant/releases/download/v${OCTANT_VERSION}/octant_${OCTANT_VERSION}_Linux-64bit.tar.gz \
-    && tar -xvf octant.tar.gz \
-    && mv octant_${OCTANT_VERSION}_Linux-64bit/octant /usr/local/bin/ \
-    && rm -rf octant_${OCTANT_VERSION}_Linux-64bit
-
```

Comment:
I removed anything related to Octant and we should do the same in other devcontainers (such as in the API repo). We are using `k9s` nowadays which is lighter and more powerful.